db/crud.go

```diff
@@ -635,6 +642,18 @@ func (db *Database) ImportDoc(docid string, body Body, isDelete bool) (docOut *d
 			return nil, nil, base.ErrAlreadyImported
 		}
 
+		// If there's a cas mismatch, the doc has been updated since the version that triggered the import.  Handling depends on import mode
+		if doc.Cas != importCas {
```

Comment:
// At the point we know it's an SDK write (as opposed to SG write), since we checked if SG write above